libvast/src/system/exporter.cpp

```diff
@@ -75,7 +75,12 @@ void ship_results(exporter_actor::stateful_pointer<exporter_state> self) {
       return;
     }
     for (auto& t : *transformed)
-      self->anon_send(st.sink, std::move(t));
+      self->request(st.sink, caf::infinite, std::move(t))
```

Comment:
One of the receivers depends on knowin who the sender is. In CAF 0.17.6 the anon_send didn't work properly and the sender was available. In CAF 0.18.6 the sender is nullptr so the code stopped working.